docs/scos/dev/technology-partner-guides/202108.0/payment-partners/adyen/integrating-adyen.md

```diff
@@ -25,157 +25,275 @@ There is currently an issue when using giftcards with Adyen. Our team is develop
 
 {% endinfo_block %}
 
-## Integration into a Project
+This article provides step-by-step instructions on integrating the Adyen module into your project.
 
-Add sub form plugins and payment method handlers:
+## Prerequisites
 
-**\Pyz\Yves\CheckoutPage\CheckoutPageDependencyProvider:**
+Prior to integrating Adyen into your project, make sure you [installed and configured the Adyen module](/docs/scos/user/technology-partners/{{page.version}}/payment-partners/adyen/adyen-installation-and-configuration.html).
+
+## Project integration
+
+To integrate Adyen, do the following:
+
+### Data import
+
+Import payment methods to be able to display them:
+
+**data/import/common/common/payment_method.csv**
+
+```yaml
+payment_method_key,payment_method_name,payment_provider_key,payment_provider_name,is_active
+adyenCreditCard,Adyen Credit Card,Adyen,Adyen,1
+adyenSofort,Adyen Sofort,Adyen,Adyen,1
+adyenDirectDebit,Adyen Direct Debit,Adyen,Adyen,1
+adyenKlarnaInvoice,Adyen Klarna Invoice,Adyen,Adyen,1
+adyenPrepayment,Adyen Prepayment,Adyen,Adyen,1
+adyenIdeal,Adyen Ideal,Adyen,Adyen,1
+adyenPayPal,Adyen PayPal,Adyen,Adyen,1
+adyenAliPay,Adyen AliPay,Adyen,Adyen,1
+adyenWeChatPay,Adyen WeChatPay,Adyen,Adyen,1
+```
+
+**data/import/common/DE/payment_method_store.csv**
+
+```yaml
+payment_method_key,store
+adyenCreditCard,DE
+adyenCreditCard,DE
+adyenSofort,DE
+adyenDirectDebit,DE
+adyenKlarnaInvoice,DE
+adyenPrepayment,DE
+adyenIdeal,DE
+adyenPayPal,DE
+adyenAliPay,DE
+adyenWeChatPay,DE
+```
+
+### OMS
+
+{% info_block infoBox "Exemplary content" %}
+
+The state machines provided below are examples of the payment service provider flow.
+
+{% endinfo_block %}
+
+Create the OMS:
+
+1. Copy this state machines on the project level and adjust them according to your requirements:
+
+<details>
+<summary>src/Pyz/Zed/Oms/OmsDependencyProvider.php</summary>
```

Comment:
@lenadoc I think the file is `config_default.php` not `src/Pyz/Zed/Oms/OmsDependencyProvider.php`